tutorials/create-cloud-build-image-factory-using-packer/windows/scripts/cleanup-packer.ps1

```diff
@@ -0,0 +1,101 @@
+$machinePath = [Microsoft.Win32.Registry]::LocalMachine.OpenSubKey('SYSTEM\CurrentControlSet\Control\Session Manager\Environment\').GetValue('PATH', '', [Microsoft.Win32.RegistryValueOptions]::DoNotExpandEnvironmentNames).ToString()
+    
+@"
+    User PATH:
+    $userPath
+    
+    Machine PATH:
+    $machinePath
+    
+"@ | Out-File "C:\PATH_backups_ChocolateyUninstall.txt" -Encoding UTF8 -Force
+    
+    if ($userPath -like "*$env:ChocolateyInstall*") {
+    Write-Output "Chocolatey Install location found in User Path. Removing..."
+    # WARNING: This could cause issues after reboot where nothing is
+    # found if something goes wrong. In that case, look at the backed up
+    # files for PATH.
+    [System.Text.RegularExpressions.Regex]::Replace($userPath, [System.Text.RegularExpressions.Regex]::Escape("$env:ChocolateyInstall\bin") + '(?>;)?', '', [System.Text.RegularExpressions.RegexOptions]::IgnoreCase) | %{[System.Environment]::SetEnvironmentVariable('PATH', $_.Replace(";;",";"), 'User')}
+    }
+    
+    if ($machinePath -like "*$env:ChocolateyInstall*") {
+    Write-Output "Chocolatey Install location found in Machine Path. Removing..."
+    # WARNING: This could cause issues after reboot where nothing is
+    # found if something goes wrong. In that case, look at the backed up
+    # files for PATH.
+    [System.Text.RegularExpressions.Regex]::Replace($machinePath, [System.Text.RegularExpressions.Regex]::Escape("$env:ChocolateyInstall\bin") + '(?>;)?', '', [System.Text.RegularExpressions.RegexOptions]::IgnoreCase) | %{[System.Environment]::SetEnvironmentVariable('PATH', $_.Replace(";;",";"), 'Machine')}
+    }
+    
+    # Adapt for any services running in subfolders of ChocolateyInstall
+    $agentService = Get-Service -Name chocolatey-agent -ErrorAction SilentlyContinue
+    if ($agentService -and $agentService.Status -eq 'Running') { $agentService.Stop() }
+    # TODO: add other services here
+    
+    # delete the contents (remove -WhatIf to actually remove)
+    Remove-Item -Recurse -Force "$env:ChocolateyInstall" -WhatIf
+    
+    [System.Environment]::SetEnvironmentVariable("ChocolateyInstall", $null, 'User')
+    [System.Environment]::SetEnvironmentVariable("ChocolateyInstall", $null, 'Machine')
+    [System.Environment]::SetEnvironmentVariable("ChocolateyLastPathUpdate", $null, 'User')
+    [System.Environment]::SetEnvironmentVariable("ChocolateyLastPathUpdate", $null, 'Machine')
+}
+
+function Remove-PackerUser{
+    <#
+        .SYNOPSIS
+            This removes the local packer_user account used for packer winRM connection
+    #>
+    param(
+        [String] $userAccount # default, packer_user
+    )
+    Write-Output "+++ Removing local user account for packer +++"
+    Remove-LocalUser -Name $userAccount
+}
+
+function Remove-WinRMConfig {
```

Comment:
See other comment, I do not think you need this

Reply:
See [response](https://github.com/GoogleCloudPlatform/community/pull/1260#discussion_r430101384). Packer's best practices is to set up a HTTPS listener for WinRM, which is what the bootstrap script does